src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,40 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public int compareTo(Name otherName){
+        int i = 0;
```

Comment:
While it is good that you can implement this. Why not simply use String.complareTo() method?

Reply:
"A".compareTo("a") will return a negative value.
But I want capital letters to be displayed after lowercase letters